backend/src/services/schema.ts

```diff
@@ -47,3 +48,15 @@ export async function addDefaultSchemas() {
     await createSchema({ ...schema, use: SchemaType.DEPLOYMENT }, true)
   }
 }
+
+export async function getManagerAndReviewer(metadata: ModelMetadata) {
+  const schema = await findSchemaByRef(metadata.schemaRef)
+
+  if (!schema) {
+    return { managerTitle: 'Managers', reviewerTitle: 'Technical Reviewer' }
+  }
+
+  const managerTitle = schema.schema.properties.contacts.properties.manager.title
```

Comment:
Schema is typed as `any`, but we're accessing numerous properties unsafely here. Either:
1. If we know these properties will always exist, update the type from `any` to accurately reflect what we know about `schema`
2. If we can't guarantee that these properties will always exist, update this code to check each property exists before accessing it. Also include fallback logic.

Reply:
This work has been spun out into the following ticket: BAI-854